### ISSUE TITLE
Running in Parallel

### ISSUE THREAD

**Issue by smart-patrol** [tnq177/transformers_without_tears#1]
Looks like the implementation is for single GPU.

Tried simply wrapping `model = torch.nn.DataParallel(model)`

and got back a nasty logit mask does not match index error.

Anyway to run this across multiple gpus?

**Comment by tnq177** [comment]
No. And I don’t plan to support it. Every details in this paper/code could be easily applied to fairseq which is fast and supports multiple gpus.
